### PR TITLE
feat: add OpsGenie alert levels to payload

### PR DESCRIPTION
**Pull request by OliverNChalk**
##### Summary

From reading [this comment](https://github.com/netdata/netdata/issues/9858#issuecomment-687116258) I was expecting the opsgenie integration to correctly set the level based on the alert severity. However, it does not set this property which results in all alerts defaulting to priority P3.

##### Test Plan

This is a minor change that **I HAVE NOT TESTED** as I am not well setup to build from source. If necessary I can attempt to setup a dev environemnt, however, hopefully someone from the team can quickly validate my PR & test it (provided this change is desired).

##### Additional Information

- This change affects the OpsGenie alert behavior in `health/notifications/*`.
- This will change the default priority on OpsGenie for any users of the integration. This means `WARN` level will remain the same but `CLEAR` and `CRITICAL` will change. I suspect not many people are using the current behaviour as it results in Medium alerts being created when a Netdata alert is `CLEAR`'d.

### CODE REVIEW

File: health/notifications/alarm-notify.sh.in [hunk 1]
```diff
@@ -2321,14 +2321,22 @@ EOF
...
     return 1
   fi
 
+  # Priority for OpsGenie alert
+  case "${status}" in
+  CRITICAL) priority="P5" ;; # Critical is P5
+  WARNING) priority="P3" ;; # Warning is P3
+  CLEAR) priority="P1" ;; # Clear is P1
+  *) priority="P3" ;; # OpsGenie's default alert level is P3
+  esac
```
Comment (ilyam8) [inline health/notifications/alarm-notify.sh.in:2338]: I see that the [importance level decreases from P1 to P5](https://support.atlassian.com/opsgenie/docs/what-is-the-priority-level-of-integration/).

According to [Update alert priority level](https://support.atlassian.com/opsgenie/docs/update-alert-priority-level/):

- P1 is Critical.
- P2 is High
- P3 is Moderate.
- P4 is Low.
- P5 is informational.

**Review by ilyam8 — CHANGES_REQUESTED** [review]

File: health/notifications/alarm-notify.sh.in (hunk 1 above)
Comment (OliverNChalk) [reply health/notifications/alarm-notify.sh.in:2338]: damn, that's embarassing. ive force pushed the branch with the fix. although happy if you guys implement yourselves as this PR is more of a suggestion/easier to draft than an issue

Comment (ilyam8) [reply health/notifications/alarm-notify.sh.in]: No worries, @OliverNChalk. Your changes look good, thanks 👍

**Review by ilyam8 — DISMISSED** [review]

File: health/notifications/alarm-notify.sh.in [hunk 2]
```diff
@@ -2321,14 +2321,22 @@ EOF
...
-  local payload httpcode oldv currv
+  local payload httpcode oldv currv priority
   [ "${SEND_OPSGENIE}" != "YES" ] && return 1
 
   if [ -z "${OPSGENIE_API_KEY}" ] ; then
     info "Can't send Opsgenie notification, because OPSGENIE_API_KEY is not defined"
     return 1
   fi
 
+  # Priority for OpsGenie alert
```
Comment (ilyam8) [inline health/notifications/alarm-notify.sh.in:2332]:
```suggestion
  # Priority for OpsGenie alert (https://support.atlassian.com/opsgenie/docs/update-alert-priority-level/)
```

**Review by ilyam8 — APPROVED** [review]

**Review by MrZammler — APPROVED** [review]

**Comment by ilyam8** [comment]
Thanks, @OliverNChalk!

**Pull request merged**